### PR TITLE
Changed Capping Analytics FAQ and minor changes

### PR DESCRIPTION
**Pull request by marksou**
Changed the Capping FAQ to include size based cap as per Freshdesk # 1318. Plus a couple of minor changes.

### CODE REVIEW

File: tyk-docs/content/frequently-asked-questions/capping-analytics-data-storage.md [hunk 1]
```diff
@@ -51,4 +53,32 @@ Set the data expires to a time in seconds for it to expire. Tyk will calculate t
     "enforce_org_quota": false
 ```
 
-> **Note**: This will only work for v2.2.0.23, if you are running an earlier patch, you will need to `enforce_org_quota` set to `true`.
\ No newline at end of file
+> **Note**: This will only work for v2.2.0.23, if you are running an earlier patch, you will need to `enforce_org_quota` set to `true`.
+
+## <a name="size-based"></a> Size Based Cap
+
+### Step 1: Run tyk_analytics
```
Comment (buger) [inline tyk-docs/content/frequently-asked-questions/capping-analytics-data-storage.md]: "Run" is not a right word here. I propose to unite step1 and step2 into one.

File: tyk-docs/content/frequently-asked-questions/capping-analytics-data-storage.md [hunk 2]
```diff
@@ -51,4 +53,27 @@ Set the data expires to a time in seconds for it to expire. Tyk will calculate t
...
\ No newline at end of file
+> **Note**: This will only work for v2.2.0.23, if you are running an earlier patch, you will need to `enforce_org_quota` set to `true`.
+
+## <a name="size-based"></a> Size Based Cap
+
+### Step 1: Add the Size Cap
+
+>  **Note**: The size value should be in bytes, and we recommend using a value just under the amount of RAM on your machine.
+
+Run this command in your preferred MongoDB tool:
```
Comment (buger) [inline tyk-docs/content/frequently-asked-questions/capping-analytics-data-storage.md]: Replace ` preferred MongoDB tool` by `MongoDB shell`. Some visual mongodb tools do not have shell.

File: tyk-docs/content/frequently-asked-questions/capping-analytics-data-storage.md [hunk 3]
```diff
@@ -51,4 +53,27 @@ Set the data expires to a time in seconds for it to expire. Tyk will calculate t
...
+
+```{.copyWrapper}
+    use tyk_analytics
+    db.runCommand({"convertToCapped": "tyk_analytics", size: 100000});
+```
+
+### Step 2: Run again on the Secondary Analytics Log
```
Comment (buger) [inline tyk-docs/content/frequently-asked-questions/capping-analytics-data-storage.md]: It is not a secondary log. More like differences in configuration. If the standard on-premise version is used, there will be only `tyk_analytics` collection. If person is using "mongo_selective" pump, which stores data per organization, collection name will be "z_tyk_analyticz_" + <org-id>. 

So it is not step-1 and step-2. Just different setups have different collection names.

Comment (marksou) [reply tyk-docs/content/frequently-asked-questions/capping-analytics-data-storage.md]: Changed and pushed.

File: tyk-docs/content/frequently-asked-questions/capping-analytics-data-storage.md [hunk 4]
```diff
@@ -51,4 +53,30 @@ Set the data expires to a time in seconds for it to expire. Tyk will calculate t
...
+    use tyk_analytics
+    db.runCommand({"convertToCapped": "tyk_analytics", size: 100000});
+```
+
+### Adding the Size Cap if using a mongo_selective Pump
+
+The `mongo_selective` pump stores data on a per organisation basis. You will have to run the following command for an individual organisation as follows.
+
+
+>  **Note**: The `LogId` will be a long number.
```
Comment (buger) [inline tyk-docs/content/frequently-asked-questions/capping-analytics-data-storage.md]: It should be `OrgID`, and it is not a number but a alpha-numeric ID. Probably make sense remove this <note> at all, and in code sample, change collection name look like: `z_tyk_analyticz_<org-id>`

Comment (marksou) [reply tyk-docs/content/frequently-asked-questions/capping-analytics-data-storage.md]: Done.

**Comment by buger** [comment]
👍

**Pull request merged**